packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -377,13 +393,46 @@ export default class TableResize implements EditorPlugin {
         }
     }
 
-    private createResizer(
+    private setTableResizer(rect: null): void;
+    private setTableResizer(rect: Rect): void;
+    private setTableResizer(rect: Rect | null): void {
+        // remove old one if exists
+        while (this.tableResizerContainer.hasChildNodes()) {
+            this.tableResizerContainer.removeChild(this.tableResizerContainer.lastChild);
+        }
+        this.tableResizer = null;
```

Comment:
remove event handler?

Reply:
Yes, the event handler is still onMouseMove(mousemove) and we already removed it in endResizeCells code.